src/main/java/com/github/pascalgn/jiracli/Jiracli.java

```diff
@@ -70,24 +70,30 @@ public static void main(String[] args) {
             System.out.println("  -g, --gui       show a graphical console window");
             System.out.println("  -c, --console   run in console mode, using stdin and stdout");
             System.out.println("  -V, --version       show the program version and exit");
+            System.out.println("  --command <line>    execute the line as a command");
         } else if (options.get(Option.VERSION) == Boolean.TRUE) {
             System.out.println(Constants.getTitle());
         } else {
             LOGGER.debug("Starting {}...", Constants.getTitle());
 
-            final boolean gui;
-            if (options.get(Option.CONSOLE) == Boolean.TRUE) {
-                gui = false;
-            } else if (options.get(Option.GUI) == Boolean.TRUE) {
-                gui = true;
+            if (options.get(Option.CMD_MODE) == Boolean.TRUE) {
+                executeCommand((String)options.get(Option.CMD_LINE));
```

Comment:
I think this can be moved up on the same level as the `options.get(Option.VERSION) == Boolean.TRUE` check

Reply:
Have moved command option check one level up.